invenio/base/scripts/database.py

```diff
@@ -86,6 +86,9 @@ def init(user='root', password='', yes_i_know=False):
             (cmd_prefix + '-e "GRANT ALL PRIVILEGES ON '
              '{CFG_DATABASE_NAME}.* TO {CFG_DATABASE_USER}@localhost '
              'IDENTIFIED BY {CFG_DATABASE_PASS}"'),
+            (cmd_prefix + '-e "GRANT ALL PRIVILEGES ON '
+             '{CFG_DATABASE_NAME}.* TO {CFG_DATABASE_USER}@\'%\' '
+             'IDENTIFIED BY {CFG_DATABASE_PASS}"'),
```

Comment:
You still have a username+password. So as an admin you should secure your MySQL instance, which includes port blocking and optional (auto-)blacklisting. I would also say that the new config is more modern because it does not assume any more that MySQL and Invenio run on the same system (even in the same container/chroot/jail/...)